common/src/main/java/com/gregtechceu/gtceu/common/data/GCyMMachines.java

```diff
@@ -0,0 +1,732 @@
+                        default -> "Simple"; // Should never be hit.
+                    } + " Parallel Control Hatch")
+                    .rotationState(RotationState.ALL)
+                    .abilities(PartAbility.PARALLEL_HATCH)
+                    .workableTieredHullRenderer(GTCEu.id("block/machines/parallel_hatch_mk" + (tier - 4)))
+                    .tooltips(Component.translatable("gtceu.parallelhatch_mk"+tier))
+                    .compassNode("parallel_hatch")
+                    .register(),
+            IV, LuV, ZPM, UV);
+
+    public final static MultiblockMachineDefinition LARGE_MACERATION_TOWER = REGISTRATE.multiblock("large_maceration_tower", WorkableElectricMultiblockMachine::new)
+            .langValue("Large Maceration Tower")
+            .tooltips(Component.translatable("gtceu.multiblock.parallelizable.tooltip"))
+            .tooltips(Component.translatable("gtceu.machine.available_recipe_map_1.tooltip", Component.translatable("gtceu.recipemap.macerator")))
+            .rotationState(RotationState.NON_Y_AXIS)
+            .recipeType(MACERATOR_RECIPES)
+            .recipeModifier(GTRecipeModifiers.PARALLEL_HATCH.apply(OverclockingLogic.PERFECT_OVERCLOCK, GTRecipeModifiers.ELECTRIC_OVERCLOCK))
+            .appearanceBlock(CASING_SECURE_MACERATION)
+            .pattern(definition -> FactoryBlockPattern.start()
+                    .aisle("XXXXX", "XXXXX","XXXXX", "XXXXX")
+                    .aisle("XXXXX", "XGGGX","XGGGX", "X###X")
+                    .aisle("XXXXX", "XGGGX","XGGGX", "X###X")
+                    .aisle("XXXXX", "XGGGX","XGGGX", "X###X")
+                    .aisle("XXXXX", "XXXXX","XXSXX", "XXXXX")
+                    .where('S', controller(blocks(definition.get())))
+                    .where('X', blocks(CASING_SECURE_MACERATION.get()).setMinGlobalLimited(55)
+                            .or(Predicates.autoAbilities(definition.getRecipeTypes()))
+                            .or(Predicates.autoAbilities(true, false, true)))
+                    .where('G', Predicates.blocks(CRUSHING_WHEELS.get()))
+                    .where('#', Predicates.air())
+                    .build())
+            .workableCasingRenderer(GTCEu.id("block/casings/gcym/secure_maceration_casing"),
+                    GTCEu.id("block/multiblock/gcym/large_maceration_tower"), false)
+            .compassSections(GTCompassSections.TIER[IV])
+            .compassNodeSelf()
+            .register();
+
+    public final static MultiblockMachineDefinition LARGE_CHEMICAL_BATH = REGISTRATE.multiblock("large_chemical_bath", WorkableElectricMultiblockMachine::new)
+            .langValue("Large Chemical Bath")
+            .tooltips(Component.translatable("gtceu.multiblock.parallelizable.tooltip"))
+            .tooltips(Component.translatable("gtceu.machine.available_recipe_map_2.tooltip", Component.translatable("gtceu.recipemap.ore_washer"), Component.translatable("gtceu.recipemap.chemical_bath")))
```

Comment:
these ain't the right lang keys, use the preexisting ones like `gtceu.ore_washer`, that you can get from `CHEMICAL_BATH_RECIPES.registryName.toLanguageKey()`